specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/stable/2017-04-01/azureactivedirectory.json

```diff
@@ -0,0 +1,507 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-04-01",
+    "title": "azureactivedirectory",
+    "description": "Azure Active Directory Client.",
+    "x-ms-code-generation-settings": {
+      "name": "azureactivedirectoryClient"
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "security": [
+    {
+      "azure_auth": ["user_impersonation"]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/microsoft.aadiam/operations": {
```

Comment:
I suggest adding x-ms-paggeable extension for listing: http://azure.github.io/autorest/extensions/#x-ms-pageable 
The Canonical way to use this extension is 
``` json
"x-ms-pageable":  {                         
"nextLinkName": "nextLink"                  
}
```
inside the Operation Object.